packages/cli/src/commands/contract/compile.ts

```diff
@@ -69,7 +69,7 @@ export class CompileContract extends Command {
     for (const contractName of contractNames) {
       const contractInfo = config.contracts[contractName];
       if (!contractInfo) {
-        this.error(`Cannot find contract info for ${contractName} contract in swanky.config.json`);
+        this.error(`Cannot find contract info for ${contractName} contract in swanky.config.json. If you are trying to compile all available contracts in the project, please use --all flag.`);
```

Comment:
getting `undefined` here is trivial bug which I introduced in the last PR when resolving contracts.
So, when neither contract name nor `--all` flag specified, we gets error even before here. it will be fixed with this PR https://github.com/AstarNetwork/swanky-cli/pull/126

Reply:
@shunsukew this one can be closed now?

Reply:
yes